identity/app/model/PhoneNumbers.scala

```diff
@@ -0,0 +1,15 @@
+package model
+
+object PhoneNumbers {
+
+  lazy val countryCodes = List(1, 7, 20, 27, 30, 31, 32, 33, 34, 36, 39, 40, 41, 43, 44, 45, 46, 47, 48, 49, 51, 52, 53,
```

Comment:
This doesn't look very maintainable, what happens if you've left out another country or its code? Would a Map of Country to Country Code be more maintainable?